### PR TITLE
New welding tool variant for Engineers

### PR DESCRIPTION
**Pull request by Momoberry**
## About the PR 
PR adds in a new yellow welding tool for engineers, ~~that has its capacity increased to 45u instead of 20. Hopefully helps engineers with welding pipes, walls, and atmos pipes without having to take constant trips.~~

## Why's this needed? 
Engineers need some more QOL for doing their job. ~~Engineering consistently require welding, they should be able to weld without having to constantly refill every 6 clicks, because that's not enjoyable.~~

### CODE REVIEW

**Comment by ThatFiveGuys** [comment]
45 seems really high, welders are strong weapons, should be closer to 30.

**Comment by Momoberry** [comment]
> 45 seems really high, welders are strong weapons, should be closer to 30.

I'd rather make welders weaker if I have to change stuff because of it being used for other things.

**Comment by ThatFiveGuys** [comment]
Why not just have variable fuel consumption to where hitting objects is a different amount from
hitting mobs? I think combat wise welders are in a good spot right now and id rather not see them seriously messed with

**Comment by pgmzeta** [comment]
can you give this to the engiborg too please

**Comment by Tarmunora** [comment]
Yeah, no. Engineer tools - with the exception of the RCD-C, which is head-of-staff swag - aren't better than other tool, just yellower.

**Comment by Sord213** [comment]
I dunno about yass but I wouldn't mind adding this to manufacturers with a rare ore cost requirement. Upgrade you can make if you have active miners.

File: code/obj/item/tool/weldingtool.dm [hunk 1]
```diff
@@ -342,6 +342,13 @@
 			return 1 //welding, has fuel
 		return 0 //not welding
 
+/obj/item/weldingtool/yellow
+	icon_state = "weldingtool-off-yellow"
+	item_state = "weldingtool-off-yellow"
+	icon_state_variant_suffix = "-yellow"
+	uses_multiple_icon_states = 1
+	capacity = 45
```
Comment (ZeWaka) [inline code/obj/item/tool/weldingtool.dm:350]:
```suggestion
```

**Comment by ZeWaka** [comment]
Merging for the yellow sprite, potential upgrades or special welding tool expansion methods can be addressed in a later PR.

**Pull request merged**